package.json

```diff
@@ -139,6 +139,9 @@
     "test-client": "NODE_ENV=test NODE_PATH=test:client TEST_ROOT=client test/runner.js",
     "test-server": "NODE_ENV=test NODE_PATH=test:server:client TEST_ROOT=server test/runner.js",
     "test-test": "NODE_ENV=test NODE_PATH=test:client TEST_ROOT=test test/runner.js",
+    "test-client:watch": "nodemon -e js,jsx --exec npm run test-client",
```

Comment:
`nodemon` has a [list](https://github.com/novemberborn/ignore-by-default/blob/master/index.js#L5-L10) of directories it ignores by default.  They are overridable in `nodemon` but I think they are sane.